bootstrap.sh

```diff
@@ -33,6 +26,7 @@ export TARGET_DIR=${TARGET_DIR:-$(pwd)/lib/binding}
 export OSRM_REPO=${OSRM_REPO:-"https://github.com/Project-OSRM/osrm-backend.git"}
 export OSRM_DIR=$(pwd)/deps/osrm-backend-${BUILD_TYPE}
 export JOBS=${JOBS:-1}
+export INSTALL_PREFIX=${INSTALL_PREFIX:-"/tmp/osrm-backend"}
```

Comment:
`INSTALL_PREFIX` could be confused with `TARGET_DIR`. `TMP_PREFIX` would maybe be better?